stack/.env.example

```
@@ -20,10 +20,16 @@ MUCGPT_SSO_CLIENTID=mucgpt
 MUCGPT_SSO_CLIENT_SECRET=client_secret
 MUCGPT_SSO_SCOPE=profile,openid,LHM,LHM_Extended
 MUCGPT_SSO_ROLE=lhm-ab-mucgpt-user
+# Redis
+MUCGPT_REDIS_HOST=localhost
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MUCGPT_REDIS_HOST key should go before the MUCGPT_SSO_CLIENTID key

(UnorderedKey)